dynamiq/nodes/llms/perplexity.py

```diff
@@ -10,17 +16,46 @@ class Perplexity(BaseLLM):
     Attributes:
         connection (PerplexityConnection): The connection to use for the Perplexity LLM.
         MODEL_PREFIX (str): The prefix for the Perplexity model name.
+        return_citations (bool): Whether to return citations in the response.
     """
 
     connection: PerplexityConnection
     MODEL_PREFIX = "perplexity/"
+    return_citations: bool = Field(default=False, description="Whether to return citations in the response")
```

Comment:
Please move up this attribute, or move up MODEL_PREFIX. Also could add extra blank line between them. A mix of model fields and constants looks not good.